elfi/__init__.py

```diff
@@ -6,6 +6,7 @@
 import elfi.model.tools as tools
 from elfi.client import get_client, set_client
 from elfi.methods.methods import *
+from elfi.methods.post_processing import *
```

Comment:
The importables are defined in the `__all__` variable of the module. By importing with the star changes only need to made in one place. It might be useful to add one layer by specifying all the imports of the `methods` package in `methods/__init__.py` though.